Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -493,15 +493,48 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.objectivesText] = {"Travel to the Echo Isles and use the offering at Bwonsamdi's shrine."},
             [questKeys.objectives] = nil,
         },
+        [78226] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210026}}},
+        },
+        [78227] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210043}}},
+        },
+        [78228] = { -- This quest is not in the game. It's a placeholder fake quest to make the quest chain work.
+            [questKeys.name] = "Trial of The Owls",
+            [questKeys.startedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 25,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Obtain 1 symbol of the owl to start the Trial of The Owls quest chain."},
+            [questKeys.objectives] = {nil,nil,{{210044}}},
+        },
         [78229] = {
             [questKeys.name] = "Trial of The Owls",
             [questKeys.startedBy] = {{12042}},
-            [questKeys.finishedBy] = nil,
+            [questKeys.finishedBy] = {{12042,}},
             [questKeys.requiredLevel] = 1,
             [questKeys.questLevel] = 25,
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.DRUID,
-            [questKeys.objectivesText] = {"Bring symbols of the three owls to Loganaar in Moonglade."},
+            [questKeys.objectivesText] = {"To start this quest, bring 1 symbol of the owl to Loganaar in Moonglade."},
```

Comment:
No manual edits in the base file, move this correction to the questFixes file.

Reply:
I really need to get it through my head that we don't touch the base files in anyway outside of adding direct scraper data. xD

Fixed in : https://github.com/Questie/Questie/pull/5378/commits/a85ba7acb03f879941f032824afdf66ff09c1434